gui/create-item.lua

```diff
@@ -224,16 +224,21 @@ function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, ge
  creatureID = tonumber(creatureID)
  -- get the actual raws of the target creature
  local creatorRaceRaw = df.creature_raw.find(creatureID)
+ local wholePart = false
  casteID = tonumber(casteID)
  bodypart = tonumber(bodypart)
  partlayer = tonumber(partlayer)
+ if partlayer == -1 then -- somewhat similar to the bodypart variable below, a value of -1 here means that the user wants to spawn a whole body part. we set the partlayer to 0 (outermost) becuase the specific layer isn't important, and we're spawning them all anyway.
```

Comment:
```suggestion
 if partlayer == -1 then -- somewhat similar to the bodypart variable below, a value of -1 here means that the user wants to spawn a whole body part. we set the partlayer to 0 (outermost) because the specific layer isn't important, and we're spawning them all anyway.
```